worlds/stardew_valley/options.py

```diff
@@ -238,9 +259,23 @@ class BuildingProgression(Choice):
     option_progressive_early_shipping_bin = 2
 
 
+class FestivalLocations(Choice):
+    """Locations for attending and participating in festivals
+    With Disabled, you do not need to attend festivals
+    With Easy, there are checks for participating in festivals
+    With Hard, the festival checks are only granted when the player performs well in the festival
+    """
+    internal_name = "festival_locations"
+    display_name = "Festival Locations"
+    default = 1
+    option_disabled = 0
+    option_easy = 1
+    option_hard = 2
+
+
 class ArcadeMachineLocations(Choice):
     """How are the Arcade Machines handled?
-    Vanilla: The arcade machines are not included in the Archipelago shuffling.
+    Disabled: The arcade machines are not included in the Archipelago shuffling.
```

Comment:
May be worth noting that Let's Play A Game can still be required. Or that could be changed so it isn't an option if Arcade Machines are disabled.